packages/noya-designsystem/src/components/GridView.tsx

```diff
@@ -0,0 +1,103 @@
+import { memo, useCallback, ReactNode } from 'react';
+import styled from 'styled-components';
+
+const Grid = styled.div(({ theme }) => ({
+  flex: 1,
+  color: theme.colors.text,
+  display: 'grid',
+  padding: '14px',
+  gridTemplateColumns: 'repeat(auto-fill, 200px)',
+  gridTemplateRows: 'repeat(auto-fill, 180px)', // Makes it go
+  justifyContent: 'space-between',
+  gap: '20px',
+  overflowY: 'scroll',
+}));
+
+const ScrollArea = styled.div(({ theme }) => ({
+  flex: '1 1 0px', // Ignore the grid's intrinsic height
+  display: 'flex',
+  overflowY: 'auto',
+  flexDirection: 'column',
+}));
+
+const Container = styled.div(({ theme }) => ({
+  flex: '1',
+  display: 'flex',
+  flexDirection: 'column',
+}));
+
+const ItemContainer = styled.div<{ selected: boolean }>(
+  ({ theme, selected }) => ({
+    width: '200px',
+    height: '130px',
+    backgroundColor: theme.colors.sidebar.background,
+    alignItems: 'center',
+    justifyContent: 'center',
+    display: 'flex',
+    borderRadius: '12px',
+    cursor: 'pointer',
+    border: `2px ${selected ? 'solid' : 'none'} rgb(132,63,255)`,
+  }),
+);
+
+const SwatchContainer = styled.div(({ theme }) => ({
+  heigth: '180px',
+}));
+
+const ItemTitle = styled.span(({ theme }) => ({
+  color: theme.colors.textDecorativeLight,
+  margin: '0px',
```

Comment:
Does margin 0 do anything here?

Reply:
it did when it was a p, but not needed anymore with the new layout. You're right